src/main/java/cam72cam/mod/gui/helpers/GUIHelpers.java

```diff
@@ -109,6 +109,18 @@ public static void drawTankBlock(int x, int y, int width, int height, Fluid flui
         }
     }
 
+    /** Draw a left-aligned shadowed string */
+    public static void drawString(String text, int x, int y, int color) {
+        drawString(text, x, y, color, new Matrix4());
+    }
+    public static void drawString(String text, int x, int y, int color, Matrix4 matrix) {
+        RenderState state = new RenderState().color(1, 1, 1, 1).alpha_test(true);
+        state.model_view().multiply(matrix);
+        try (With ctx = RenderContext.apply(state)) {
+            Minecraft.getMinecraft().fontRenderer.drawString(text, x, y, color);
```

Comment:
Similar to drawCenteredString, can you include `GlStateManager.color(1, 1, 1, 0);`?  It's there to work around a weird interaction between GlStateManager and the rest of the code.